src/event_handler.py

```diff
@@ -1,12 +1,20 @@
 import boto3
 import os
+import json
+from psycopg2 import IntegrityError
+
+from src.db_helper import RunInTransaction, create_db_connection
 from src.event_mapper import event_from_json
 
+# Postgres Error codes listed at https://www.postgresql.org/docs/9.3/static/errcodes-appendix.html
+POSTGRES_UNIQUE_VIOLATION_ERROR_CODE = '23505'
```

Comment:
I'm really surprised we have to code this ourselves - doesn't the library or the exception or _something_ encode this?  Seems very 1990s coding to have to hard code an error code.

Reply:
good point.
I tried and failed to find something before, but I've tried a bit harder now and found an `errorcodes.py` module within pscopg2, so I'll add a commit to use that instead.